api/tests/Api/CampCollaborations/ListCampCollaborationsTest.php

```diff
@@ -108,12 +108,4 @@ public function testListCampCollaborationsFilteredByCampPrototypeIsAllowedForUnr
             ['href' => $this->getIriFor('campCollaboration1campPrototype')],
         ], $response->toArray()['_links']['items']);
     }
-
-    public function testSqlQueryCount() {
```

Comment:
Why this removal?

Reply:
See https://github.com/ecamp/ecamp3/pull/6834/commits/717c4ff3809fd24bf90a6739e112c8c6e1d08059